tests/tween_test.py

```diff
@@ -26,3 +35,44 @@ def test_zipkin_tween_sampling(
     assert tween.zipkin_tween(handler, None)(dummy_request) == dummy_response
     assert handler.call_count == 1
     assert mock_span.call_count == 1
+
+
+@pytest.mark.parametrize(
+    'context_stack',
+    [
+        None,
+        mock.Mock(spec=py_zipkin.stack.Stack),
+    ],
+)
```

Comment:
To be honest, the body of the test seems sufficiently different between the two cases that I would split this into two test cases. Unless you can refactor it to be (readably) similar between the two.